jfsd/main.py

```diff
@@ -106,6 +110,8 @@ def main(
         Array of external forces (N,3)
     constant_applied_torques:
         Array of external torques (N,3)
+    HIs_flag:
```

Comment:
Ideally, this flag would be a boolean. Is there a reason to use integers?

Reply:
In future it will be possible to switch among 4 different hydrodynamic descriptions, thus the integer type.

Reply:
Perhaps an Enum type would be more suitable then?

https://docs.python.org/3/library/enum.html